include/pybind11/pybind11.h

```diff
@@ -2071,21 +2071,22 @@ error_already_set::~error_already_set() {
     }
 }
 
-inline function get_type_overload(const void *this_ptr, const detail::type_info *this_type, const char *name)  {
-    handle self = detail::get_object_handle(this_ptr, this_type);
+PYBIND11_NAMESPACE_BEGIN(detail)
+inline function get_type_override(const void *this_ptr, const type_info *this_type, const char *name)  {
+    handle self = get_object_handle(this_ptr, this_type);
     if (!self)
         return function();
     handle type = self.get_type();
     auto key = std::make_pair(type.ptr(), name);
 
-    /* Cache functions that aren't overloaded in Python to avoid
+    /* Cache functions that aren't overridden in Python to avoid
        many costly Python dictionary lookups below */
-    auto &cache = detail::get_internals().inactive_overload_cache;
+    auto &cache = get_internals().inactive_override_cache;
     if (cache.find(key) != cache.end())
         return function();
 
-    function overload = getattr(self, name, function());
-    if (overload.is_cpp_function()) {
+    function override = getattr(self, name, function());
+    if (override.is_cpp_function()) {
```

Comment:
I thought it was. If I understand correctly, this check whether there is a Python override with that name. If the Python override is found, it could be an override done from pybind11, and be a `cpp_function`, so rather than wrapping a wrapper, the wrapper is unwrapped.

Can it still be overloaded? Yes, but I thínk that `cpp_function` itself is just the Python object and takes care of actual overloading by having multiple reference to C++ functions inside. It's `cpp_function::dispatcher` that does actually chooses the C++ overload; so 1 function from the Python `getattr` side.

So, I do think this is getting the override and not the overload?